feature/account/src/main/java/com/aibles/account/data/remote/RegisterService.kt

```diff
@@ -0,0 +1,13 @@
+package com.aibles.account.data.remote
+
+import com.aibles.account.data.remote.dto.RegisterRequest
+import com.aibles.account.data.remote.dto.RegisterResponse
+import retrofit2.http.Body
+import retrofit2.http.POST
+import com.aibles.finance.data.remote.util.Resource
+
+
+interface RegisterService {
+    @POST("/register")
```

Comment:
path này đã match với requirement bên BE chưa nhỉ e? Hình như docs bên BE gửi nó còn 1 vài path ở đầu nữa